src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1267,7 +1267,7 @@
     <!-- Converted switch expression must have a type, even if that's `void` for a statement. -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
     <!-- Natural type is preserved for the purpose of semantic model. Can be `null`. -->
-    <Field Name="NaturalTypeOpt" Type="TypeSymbol" Null="allow"/>
+    <Field Name="NaturalTypeOpt" Type="TypeSymbol?" Null="allow"/>
```

Comment:
It feels like a nullable annotation should be used for any reference type field with `Null="allow"`. Do we have a tracking issue for updating the generator/BoundNodes.xml to do this?

Reply:
Not that I know if.